### PR TITLE
Change redisfailover instructions

### PR DESCRIPTION
**Pull request by bbobrov**
Redisfailover is now deployed as a subchart of harbor-helm. The
instructions are updated accordingly

### CODE REVIEW

File: adoc/Private-Registry-Installation.adoc [hunk 1]
```diff
@@ -589,108 +589,58 @@ kubectl create namespace redis-operator
...
 +
-[source,bash]
-----
-kubectl -n registry create secret generic redis-auth --from-literal=password="<password-value>"
-----
-
-... Install `RedisFailover` object:
-+
-The Redis HA configuration needs to be specified as a `RedisFailover` {kube} CRD object.
+The Redis HA configuration needs to be specified in the `redisfailover` section of harbor-values.yaml.
```
Comment (lproven) [inline adoc/Private-Registry-Installation.adoc:594]:
```suggestion
The Redis HA configuration needs to be specified in the `redisfailover` section of `harbor-values.yaml`.
```

File: adoc/Private-Registry-Installation.adoc [hunk 2]
```diff
@@ -589,108 +589,58 @@ kubectl create namespace redis-operator
...
+  external:
+    addr: rfs-harbor-redisfailover:26379
+    sentinelMasterSet: mymaster // <1>
 ----
-... Wait until all the objects are prepared, specifically `rfs-harbor-redis`, the Sentinel deployment:
+<1> `mymaster` is a predefined value of redisfailover deployment and cannot be changed.
+
+... (Optional) Set up own password
++
+By default, if no secret and password is provided, the {spr} Helm chart will protect the deployed redis with a generated password. Custom password can also be provided:
```
Comment (lproven) [inline adoc/Private-Registry-Installation.adoc:622]:
```suggestion
By default, if no secret and password are provided, the {spr} Helm chart will protect the deployed redis with a generated password. A custom password can also be provided:
```

Comment (lproven) [reply adoc/Private-Registry-Installation.adoc:622]: What does a "generated password" mean here? What/who generates it?

File: adoc/Private-Registry-Installation.adoc [hunk 3]
```diff
@@ -589,108 +589,58 @@ kubectl create namespace redis-operator
...
+    addr: rfs-harbor-redisfailover:26379
     sentinelMasterSet: mymaster
-    password: securepassword // <1>
+    password: <password-value>
 ----
-<1> Replace the value for password key with the password prepared in <<install-redis-operator>>.
+
+... (Optional) Configure Redisfailover deployment
++
+By default, the Redisfailover deployment has 3 sentinel replicas, 3 redis replicas and will keep the data when the Helm chart is uninstalled. This behavior can be configured in the `redisfailover` section. See `harbor/charts/redisfailover/Values.yaml` for all configuration options.
```
Comment (lproven) [inline adoc/Private-Registry-Installation.adoc:642]:
```suggestion
By default, the Redisfailover deployment has three sentinel replicas, three redis replicas, and will keep the data when the Helm chart is uninstalled. This behavior can be configured in the `redisfailover` section. See `harbor/charts/redisfailover/Values.yaml` for all configuration options.
```

**Review by lproven — CHANGES_REQUESTED** [review]
Thanks for this! I have a few small questions and changes about the changed material before I merge it, then I will do a copy-edit in there so I do not both you with multiple requests.

File: adoc/Private-Registry-Installation.adoc (hunk 2 above)
Comment (bbobrov) [reply adoc/Private-Registry-Installation.adoc]: thanks, i have changed this line

**Comment by chabowski** [comment]
@bbobrov @lproven this PR is still open - can you please have a look - seems there is still a change requested?

**Review by lproven — APPROVED** [review]

**Comment by chabowski** [comment]
@bbobrov we have a conflict with the file - line 606-670. Seems that content has been changed upon an older version of the paper, not in synch with the repo master or so. Would you mind having a look at the content and see what to keep? Thanks very much!

**Comment by bbobrov** [comment]
done, sorry for the cryptic commit message

**Pull request merged**